### PR TITLE
Reverting RecentlyActive dimension criteria

### PR DESCRIPTION
**Pull request by robotmaxtron**
Reverting changes introduced in https://github.com/openshift/managed-prometheus-exporter-ebs-iops-reporter/pull/37 
Adding reduced Period filter to 3 hours.

### CODE REVIEW

**Comment by robotmaxtron** [comment]
Seems to be working in my stage cluster

**Comment by mh21** [comment]
Hi @robotmaxtron, this PR does not solve the the problem (as opposed to the original PR) of reducing the number of GetMetricData API calls. Running the code from the PR in our cluster, we get

```
2021-09-21 04:16:41,466 DEBUG:root:Have 73 ACTIVE_VOLUMES, seen 73 volumes, total volumes from list_metrics 37101
```

Setting the log level to debug and counting the number of GetMetricData API queries/responses (where afaict each API call queries 100 values, i.e. each of the API queries costs 0.01 * 100 / 1000 = $0.001)

```
$ grep -c '<GetMetricDataResp' < logfile
372
```
So this is $0.37 every 5 minutes, or $107 a day.

The number of API queries seems to be unchanged.

File: monitor/main.py [hunk 1]
```diff
@@ -88,7 +88,7 @@ def collect(aws):
                         }
                     ]
                 },
-                'Period': CLOUDWATCH_PERIOD * 60,
+                'Period': CLOUDWATCH_PERIOD * 36,
```
Comment (lisa) [inline monitor/main.py:91]: For future reference:

* CLOUDWATCH_PERIOD * 60 = 5 minutes * 60 = 300 minutes / 60 minutes per hour = 5 hours
* CLOUDWATCH_PERIOD * 36 = 5 minutes * 36 = 180 minutes / 60 minutes per hour = 3 hours

**Review by lisa — APPROVED** [review]
/lgtm
/approve

**Pull request merged**